content/en/blog/2020/show-source-ip/index.md

```diff
@@ -0,0 +1,332 @@
+---
+title: How to enable proxy protocol on AWS NLB and Istio ingress gateway
+subtitle: How to enable proxy protocol on AWS NLB and Istio ingress gateway
+description: How to enable proxy protocol on AWS NLB and Istio ingress gateway.
+publishdate: 2020-11-18
+attribution: "Xinhui Li (Salesforce) "
+keywords: [trafficManagement,protocol extending]
+---
+
+This blog unveil my latest experience about how to configure/enable proxy protocol with stack of AWS NLB and Isio Ingress gateway. The **Proxy Protocol** was designed to chain proxies / reverse-proxies without losing the client information with the benefits from protocol agnostic, no need change infrastructure, no negative impact from nat-ing firewalls and good scalability.  To get client IP easy to read,  it is also enabled `X-Forwarded-For` of HTTP header in the deployment. In this blog, traffic management of istio ingress are shown with httpbin service on 80 and 443 respectively to demonstrate the enabling result. Below comes a part of the protocol describing about the difference of proxy protocol v1 and v2.  In out use case, both v1 and v2 works for the purpose but AWS NLB today only support v2 shown by image below. So proxy protocol v2 is referred in the rest of this blog by default.
+
+{{< text yaml >}}
+A receiver may be configured to support both version 1 and version 2 of the
+protocol. Identifying the protocol version is easy :
+    - if the incoming byte count is 16 or above and the 13 first bytes match
```

Comment:
nit: /s/and the 13 first bytes match/the first 13 bytes match

Reply:
done, thanks